python/revng/cli/revng.py

```diff
@@ -15,14 +17,14 @@ def extend_list(paths, new_items):
 
 # flake8: noqa: F401
 def run_revng_command(arguments, options: Options):
-    # Import built-in commands
-    from .daemon import DaemonCommand
-    from .lift import LiftCommand
-    from .llvm_pipeline import IRPipelineCommand
-    from .opt import IROptCommand
-    from .override_by_name import ModelOverrideByName
-    from .pipeline import PipelineCommand
-    from .translate import TranslateCommand
+    with os.scandir(Path(__file__).parent / "_commands") as scan:
+        for entry in scan:
+            entry_path = Path(entry.path)
+            if entry.is_file():
+                import_module(f"._commands.{entry_path.stem}", "revng.cli")
+            elif entry.is_dir():
+                import_module(f"._commands.{entry_path.name}", "revng.cli")
```

Comment:
At this point we could also consider dropping `commands_registry.register_command(IRPipelineCommand())` and automatically register all those that inherit `Command`, but that's maybe too much, next time.

Reply:
Done